tests/test_regions.py

```diff
@@ -7,14 +7,13 @@
 def test_regions(ds9_region_file):
     regions = load_regions(ds9_region_file)
 
-    c1 = SkyCoord(299.8452923, 40.7403572, frame='fk5', unit='deg')
-    assert regions[0].center.position_angle(c1) == Angle(0, unit='rad')
+    assert len(regions) == 3
```

Comment:
@o-smirnov Can I get a quick eye-ball on the test case here? I just removed the `position_angle` test -- something has probably changed in the upstream astropy calculation but the actual SkyCoord value matches. We don't use `position_angle` in the code anywhere so I don't think this is a critical failure, more something

Reply:
I used `position_angle` as the comparison criteria for whatever reason in the depths of time.

Reply:
Nah, it was always a misguided test. PA is completely undefined when two positions match -- it just happened to return 0 but is allowed to return any value. The new tests are valid.